packages/strapi-design-system/src/Status/__tests__/Status.spec.tsx

```diff
@@ -28,8 +28,9 @@ describe('Status', () => {
 
     const textNode = queryByText('My status');
 
+    expect(textNode).not.toBeNull();
     await waitFor(() => {
-      expect(textNode.querySelector('div')).toBeInTheDocument();
+      expect(textNode!.querySelector('div')).toBeInTheDocument();
```

Comment:
Hmmm i'm not actually a fan of these tests at all, controversial but can you add:

```ts
// @ts-expect-error TODO: refactor these tests
```

to both the uses of `!` please? :)